packages/manager-react-components/src/components/templates/onboarding/onboarding.component.tsx

```diff
@@ -38,13 +39,14 @@ export type OnboardingLayoutProps = OnboardingLayoutButtonProps &
 const OnboardingLayoutButton: React.FC<OnboardingLayoutButtonProps> = ({
   orderButtonLabel,
   orderHref,
-  moreInfoHref,
-  moreInfoButtonLabel,
-  moreInfoIcon = ODS_ICON_NAME.externalLink,
   onOrderButtonClick,
-  onmoreInfoButtonClick,
   isActionDisabled,
   orderIam,
+  moreInfoHref,
+  moreInfoButtonLabel,
+  moreInfoButtonIcon = ODS_ICON_NAME.externalLink,
+  onmoreInfoButtonClick,
```

Comment:
```suggestion
  onMoreInfoButtonClick,
```

Reply:
Yes I put both props and deprecated this one to avoid breaking changes